docs/user-guide/src/capm3/clusterclass.md

```diff
@@ -0,0 +1,73 @@
+# Using ClusterClass with CAPM3
+
+ClusterClass is a feature of Cluster API that enables the cluster operators to create multiple clusters using a single general template. You can find detailed explanation of how to use a `ClusterClass` in the [Cluster API documentation](https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/proposals/20210526-cluster-class-and-managed-topologies.md)
```

Comment:
Line length is not (yet) enforced, but it would be appreciated if you could wrap them at 80.

Reply:
Fixed :)